substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/JfrTypeRepository.java

```diff
@@ -186,6 +190,46 @@ private static int writeClassLoaders(JfrChunkWriter writer, TypeInfo typeInfo) {
         return NON_EMPTY;
     }
 
+    private static int writeGCCauses(JfrChunkWriter writer) {
+        // GCCauses has null entries
+        GCCause[] causes = GCCause.getGCCauses();
+        int nonNullItems = 0;
+        for (int index = 0; index < causes.length; index++) {
+            if (causes[index] != null) {
+                nonNullItems++;
+            }
+        }
+
+        if (nonNullItems == 0) {
```

Comment:
Can be converted to an assertion as there must always be GC causes defined.

Reply:
Done